lib/ts/I_Layout.h

```diff
@@ -31,80 +31,93 @@
 #ifndef _I_Layout_h
 #define _I_Layout_h
 
+#include <string>
+#include "ts/string_view.h"
```

Comment:
`string_view.h` pulls in `<string>` by definition, so remove `<string> from here.